rocketc/rocketc.py

```diff
@@ -214,10 +214,9 @@ def workbench_scenarios():
              """),
         ]
 
-    @property
-    def api_rocket_chat(self):
+    def _api_rocket_chat(self):
```

Comment:
this is just a suggestion and maybe it could help for calling the api client objects: We should have a method called api_rocket_chat decorated with the property decorator (sorry for the redundancy). This method should check if the attribute "_api_rocket_chat" exists on the class; if it exists, just return it, if not, create it, assign it to the class attribute "_api_rocket_chat" and return it. In this way you can call "api_rocket_chat" method from everywhere without worrying about if the client was set or not, you're sure you will get always the api instance. Additionally you will never need to manipulate the "_api_rocket_chat" attribute